.gitignore

```diff
@@ -29,3 +29,4 @@ config/application.yml
 
 # Ignore mock mvi responses
 /config/mvi_schema/mock_mvi_responses.yml
+.idea/
```

Comment:
put it in .gitignore -- I wouldn't mind getting a license for Jetbrains RubyMine, I think that uses .idea too. Are you using IntelliJ for ruby though? The same folks make Rubymine and it might be better suited. I only used IntelliJ for java code-bases, and the only Java I've touched of late was mobile, with AndroidStudio kind of crushing it in IDE department there.